repo2docker/buildpacks/conda/install-miniconda.bash

```diff
@@ -35,6 +35,11 @@ conda config --system --add channels conda-forge
 conda config --system --set auto_update_conda false
 conda config --system --set show_channel_urls true
 
+# conda 4.7 the free channel was removed from defaults; this adds it back while 
+# maintaining correct channel priority (required for the Xeus-Cling test in 
+# test/external/reproductions.repos.yaml to pass) 
+conda config --system --set restore_free_channel true
```

Comment:
If you move this below `conda install -yq conda`, after which conda should itself be up to date, it should work.

Reply:
Indeed! Well spotted. Will see if this makes Travis happy!